src/styles/blocks/page.scss

```diff
@@ -0,0 +1,45 @@
+:root {
```

Comment:
It is better to use a separate variables.scss file to define global styles like colors and fonts. Move the global styles to a new file and import it in this file.